tests/e2e/slashing.go

```diff
@@ -255,29 +254,69 @@ func (suite *CCVTestSuite) TestHandleSlashPacketDoubleSigning() {
 	suite.Require().True(signingInfo.Tombstoned)
 }
 
-// TestHandleSlashPacketErrors tests errors for the HandleSlashPacket method in an e2e testing setting
-func (suite *CCVTestSuite) TestHandleSlashPacketErrors() {
+// TestOnRecvSlashPacketErrors tests errors for the OnRecvSlashPacket method in an e2e testing setting
+func (suite *CCVTestSuite) TestOnRecvSlashPacketErrors() {
```

Comment:
We could delete this test altogether in favor of the new unit tests that are more complete. This was my attempt at salvaging this test where possible